composer.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "josegonzalez/cakephp-version",
-    "description": "CakePHP plugin to allow versioning of records",
-    "type": "cakephp-plugin",
-    "keywords": ["cakephp", "revision", "version"],
+    "description": "Cake orm behavior to allow versioning of records",
+    "type": "library",
```

Comment:
Please do not change this. Does this fail to install in non-cakephp applications without this change?